pkg/v22/resource/clusterconfigmap/resource.go

```diff
@@ -22,11 +22,13 @@ type Config struct {
 	GetClusterConfigFunc     func(obj interface{}) (v1alpha1.ClusterGuestConfig, error)
 	GetClusterObjectMetaFunc func(obj interface{}) (metav1.ObjectMeta, error)
 	GetWorkerCountFunc       func(obj interface{}) (int, error)
+	GetSubscriptionId        func(obj interface{}, k8s kubernetes.Interface) (string, error)
```

Comment:
```suggestion
	GetSubscriptionIdFunc        func(obj interface{}, k8s kubernetes.Interface) (string, error)
```

We should follow the convention and use the Func suffix.